railgun/keps/0003-kic-kubebuilder-re-architecture.md

```diff
@@ -52,8 +64,45 @@ Historically the [Kong Kubernetes Ingress Controller (KIC)][kic] was built on ol
 
 [impl]:https://kubebuilder.io/cronjob-tutorial/controller-overview.html
 
+### Non-Goals
+
+We reference [UDPIngress][udpingress] in the implementation history (below) as it was used for demonstration, but completing new features and APIs is not in scope for this KEP, though the result of this KEP is that newer features are intended to be easier to contribute.
+
+We can use the improvements made here to _demonstrate_ the ease of adding new features on the new architecture, but the full scope of GA for new features will need to be a follow up KEP.
+
+[udpingress]:https://github.com/Kong/kubernetes-ingress-controller/milestone/14
+
 ## Proposal
 
+The historical releases of the KIC (which we will refer to as `v1.x`, even though technically that includes `v0.x` releases) were built on an older controller architecture forked from the [NGinx Ingress Controller][nginx-ingress-controller] some years prior.
```

Comment:
Maybe keep saying 
_pre-v2.0_ 
instead of
_v1.x (but keep in mind that it really means v1.x and v0.x)_?
?

Reply:
Sure sounds good!
```suggestion
The historical releases of the KIC (which we will refer to as `pre-v2`) were built on an older controller architecture forked from the [NGinx Ingress Controller][nginx-ingress-controller] some years prior.
```